docs/parameters.md

```diff
@@ -24,7 +24,8 @@ The AWS EBS CSI Driver supports [tagging](tagging.md) through `StorageClass.para
 | "volumeInitializationRate"   | integer                                           |         |  When creating a volume from a snapshot, this parameter can be used to request a provisioned initialization rate, in MiB/s.                             |
 
 ## Restrictions
-* `gp3` is currently not supported on outposts. Outpost customers need to use a different type for their volumes.
+
+* `gp3` is currently not supported on First Generation Outpost Racks. As the driver defaults to gp3 when no type is passed. Outpost customers using First Generation Outposts need to specify a different type for their volumes.
```

Comment:
Suggest rewording this bullet point to

```
* The EBS CSI Driver defaults to `gp3` volumes when no volume type is specified. If the outpost does not support `gp3` volumes, specify a supported volume type via a `StorageClass`.
```

This avoids documenting how EBS works, and thus is evergreen